sdk/core/Azure.Core/src/Pipeline/Policies/RetryPolicy.cs

```diff
@@ -200,8 +193,8 @@ private TimeSpan CalculateExponentialDelay(int attempted)
         {
             return TimeSpan.FromMilliseconds(
                 Math.Min(
-                    (1 << (attempted - 1)) * _random.Next((int)(Delay.TotalMilliseconds * 0.8), (int)(Delay.TotalMilliseconds * 1.2)),
-                    MaxDelay.TotalMilliseconds));
+                    (1 << (attempted - 1)) * _random.Next((int)(_delay.TotalMilliseconds * 0.8), (int)(_delay.TotalMilliseconds * 1.2)),
```

Comment:
It'd be great to have this in an extension method or other artifact that we could share, so that calculations for the next period were consistent across SDKs.

Reply:
I tried to extract the common logic but it seems that 

```
   return TimeSpan.FromMilliseconds(
                Math.Min(
                    (1 << (attempted - 1)) * _random.Next((int)(_delay.TotalMilliseconds * 0.8), (int)(_delay.TotalMilliseconds * 1.2)),
                    _maxDelay.TotalMilliseconds));
```

is the only part that's complicated enough to be worth sharing. Now I wonder if it's worth extracting.

Reply:
Maybe make it a static method and call this good?  Or even a private static until somebody shows a need for it elsewhere?